README.md

```diff
@@ -309,7 +264,7 @@ See [this page](./examples/main/README.md) for a full list of parameters.
 
 ### Conversation mode
 
-If you want a more ChatGPT-like experience, you can run in conversation mode by passing `-cnv` as a parameter:
+For a more ChatGPT-like experience, run `llama-cli` in conversation mode by passing `-cnv` as a parameter:
```

Comment:
I would say nowadays the "ChatGPT-like" experience is the server.

Reply:
I removed the "ChatGPT" term all together.